### PR TITLE
Fixes bug 1224151 - Allowed multiple values in graphics_devices service.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@peterbe r?

### CODE REVIEW

File: socorro/unittest/external/postgresql/test_graphics_devices.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from .unittestbase import PostgreSQLTestCase
 
 
-#==============================================================================
+# =============================================================================
```
Comment (peterbe) [inline socorro/unittest/external/postgresql/test_graphics_devices.py]: This is a larsism that is here to stay. Sure, we don't allow it in the webapp-django directory but for now I suggest you just allow it be by allowing flake8 to check and it should respect our [setup.cfg](https://github.com/mozilla/socorro/blob/master/setup.cfg).

File: socorro/external/postgresql/graphics_devices.py [hunk 2]
```diff
@@ -15,28 +15,32 @@ class GraphicsDevices(PostgreSQLBase):
...
-            ("vendor_hex", None, "str"),
-            ("adapter_hex", None, "str"),
+            ("vendor_hex", None, ["list", "str"]),
+            ("adapter_hex", None, ["list", "str"]),
         ]
         params = external_common.parse_arguments(filters, kwargs)
         for key in ('vendor_hex', 'adapter_hex'):
-            if not params[key]:
+            param = params[key]
+            if not param or (len(param) == 1 and not param[0]):
```
Comment (peterbe) [inline socorro/external/postgresql/graphics_devices.py]: What is this?! Something like `[0]` or `['']`? Why is that ever an input to this? Also why is it a problem? Why not just let the query go ahead and if nothing comes back, nothing comes back. 
And why specifically 1? What if there are others. Like `['hex1', 'hex2', '']`

File: socorro/external/postgresql/graphics_devices.py [hunk 3]
```diff
@@ -15,28 +15,32 @@ class GraphicsDevices(PostgreSQLBase):
 
     def get(self, **kwargs):
         filters = [
-            ("vendor_hex", None, "str"),
-            ("adapter_hex", None, "str"),
+            ("vendor_hex", None, ["list", "str"]),
+            ("adapter_hex", None, ["list", "str"]),
```
Comment (peterbe) [inline socorro/external/postgresql/graphics_devices.py]: Does that mean it can be either a string or a list? Or a list of strings?

**Comment by peterbe** [comment]
Some questionmarks to be discussed.

File: socorro/external/postgresql/graphics_devices.py (hunk 3 above)
Comment (AdrianGaudebert) [inline socorro/external/postgresql/graphics_devices.py]: It means it has to be a list of strings.

File: socorro/external/postgresql/graphics_devices.py (hunk 2 above)
Comment (AdrianGaudebert) [inline socorro/external/postgresql/graphics_devices.py]: This is a side effect of ``parse_arguments``, which will generate ``['']`` when the argument was empty. It should probably actually be solved in the ``parse_arguments`` function instead.

**Comment by AdrianGaudebert** [comment]
@peterbe addressed

**Comment by peterbe** [comment]
r+

**Comment by peterbe** [comment]
Actually...

I wonder if this will break the admin page where you can look up individual hexes. Care to check that first?

**Comment by AdrianGaudebert** [comment]
@peterbe That was one of the first things I verified. :)

**Pull request merged**